packages/components/src/components/Tabs/components/TabTitle/TabTitle.tsx

```diff
@@ -28,6 +33,12 @@ export const TabTitle: FC<TabTitleProps> = (props) => {
               <TunnelEntry id="ActiveTitle">
                 {p.isSelected && children}
               </TunnelEntry>
+              {status && (
+                <StatusIcon
+                  status={status}
+                  aria-label={stringFormatter.format(`tab.status.${status}`)}
```

Comment:
Hat das `StatusIcon` nicht schon eine eigene Translation für das `aria-label`?

Reply:
ja hat es aber Jan wollte gern dass in den Tabs zb "Status Fehler" statt nur "Fehler" vorgelesen wird, aber eigentlich kann ich auch einfach die Translation im Status Icpn ändern 🤔 ist glaub ich auch ok wenns immer mit "Status" drin steht